schema/request.json

```diff
@@ -30,7 +30,7 @@
     "format": {
       "type": "object",
       "patternProperties": {
-        "^jwt$|^jwt_vc$|^jwt_vp$": {
+        "^jwt$|^jose_vc$|^jose_vp$|^cose_vc$|^cose_vp$|^sdjwt_vc$|^sdjwt_vp$": {
```

Comment:
Similar question as above about including `sdjwt_vc` and `sdjwt_vp` here.

Reply:
updating to the VCDM v2 requires selecting a securing mechanism. one of those mechanisms is [VC JOSE COSE](https://www.w3.org/TR/vc-jose-cose/) which defines securing mechanisms for JOSE, COSE, and SD-JWT.

this change adds corresponding format properties for the new data model.

if you have a suggestion on how to break this into 2 PRs I'm open to it.

Reply:
@jogu does @decentralgabe's answer above satisfy you or do you want to request changes?  If you're satisfied, can you please approve?  Thanks.